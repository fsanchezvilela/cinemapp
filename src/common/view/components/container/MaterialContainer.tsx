import React from 'react';
import CssBaseline from '@material-ui/core/CssBaseline';
import Container from '@material-ui/core/Container';

export interface IMaterialContainer {
  children: React.ReactNode;
}

const MaterialContainer = ({ children }: IMaterialContainer) => (
  <>
    <CssBaseline />
    <Container maxWidth="lg">{children}</Container>
  </>
);

export default MaterialContainer;
