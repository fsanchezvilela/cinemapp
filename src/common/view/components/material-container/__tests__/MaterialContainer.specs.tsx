import React from 'react';
import { customRender } from '../../../../../../tests/test-utils';
import MaterialContainer from '../MaterialContainer';

describe('MaterialContainer', () => {
  it('Render', () => {
    const rendered = customRender(<MaterialContainer />);
    expect(rendered.container).toMatchSnapshot();
  });
  it('Should show text children container', () => {
    const rendered = customRender(
      <MaterialContainer>mock test</MaterialContainer>
    );
    expect(rendered.getByText('mock test')).not.toBeNil();
  });
});
