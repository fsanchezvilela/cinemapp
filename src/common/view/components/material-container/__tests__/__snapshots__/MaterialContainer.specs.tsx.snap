// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaterialContainer Render 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  />
</div>
`;
